bot/seasons/halloween/__init__.py

```diff
@@ -3,14 +3,8 @@
 
 class Halloween(SeasonBase):
     name = "halloween"
+    greeting = "Happy Halloween!"
     start_date = "01/10"
     end_date = "31/10"
     bot_name = "Spookybot"
-
-    def __init__(self, bot):
-        self.bot = bot
-
-    @property
-    def bot_avatar(self):
-        with open(self.avatar_path("spooky.png"), "rb") as avatar:
-            return bytearray(avatar.read())
+    icon = "/logos/logo_seasonal/halloween/spooky.png"
```

Comment:
Additional line here would be nice.
```suggestion

    icon = "/logos/logo_seasonal/halloween/spooky.png"
```

Reply:
My response in the prior comment above has a suggested edit that would add a new line as you want.